build-system/app.js

```diff
@@ -1342,4 +1337,18 @@ function generateInfo(filePath) {
       '<h3><a href = /serve_mode=cdn>Change to CDN mode (prod JS)</a></h3>';
 }
 
+function decryptDocumentKey(encryptedDocumentKey) {
+  const cryptoStart = 'ENCRYPT(';
+  if (!encryptedDocumentKey.includes(cryptoStart, 0)) {
```

Comment:
My guess is that `encryptedDocumentKey` in this context could be `null|undefined`, so this should probably be the first check.

Reply:
done